climada_petals/hazard/tc_tracks_forecast.py

```diff
@@ -401,11 +452,11 @@ def _subset_to_track(msg, index, provider, timestamp_origin, name, id_no):
                     'max_sustained_wind': ('time', np.squeeze(wnd)),
                     'central_pressure': ('time', np.squeeze(pre)/100),
                     'ts_int': ('time', timestep_int),
-                    'lat': ('time', lat),
-                    'lon': ('time', lon),
                 },
                 coords={
                     'time': timestamp,
+                    'lat': ('time', lat),
```

Comment:
@ThomasRoosli I'm only seeing this since I'm incidentally also working on this file - this change is wrong, since apparently, you cannot perform `dropna()` if lat/lon are already coordinates. See https://github.com/CLIMADA-project/climada_petals/pull/33/files#diff-f38c71c49b698e73f5039c46ceb6fe77921909d7754faa380df102339250a25cR486 for the relevant line. Do you mind if I undo this change in my current PR #36?